src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -45,51 +45,55 @@ public async Task ApplyCommand()
                 HttpContext.Response.Headers["Reached-Leader"] = "true";
 
                 var commandJson = await context.ReadForMemoryAsync(RequestBodyStream(), "external/rachis/command");
-                CommandBase command;
                 try
                 {
-                    command = CommandBase.CreateFrom(commandJson);
-                }
-                catch (InvalidOperationException e)
-                {
-                    RequestRouter.AssertClientVersion(HttpContext, e);
-                    throw;
-                }
-
-                switch (command)
-                {
-                    case AddOrUpdateCompareExchangeBatchCommand batchCmpExchange:
-                        batchCmpExchange.ContextToWriteResult = context;
-                        break;
-                    case CompareExchangeCommandBase cmpExchange:
-                        cmpExchange.ContextToWriteResult = context;
-                        break;
-                }
+                    var command = CommandBase.CreateFrom(commandJson);
```

Comment:
What happened to the `AssertClientVersion` ?

Reply:
In the 'catch' part